drv/stm32h7-fmc-demo-server/src/main.rs

```diff
@@ -216,8 +232,17 @@ fn main() -> ! {
     // space, which is fine even if it aliases (which it doesn't).
     let fmc = unsafe { &*device::FMC::ptr() };
 
+    let net = Net::from(NET.get_task_id());
+
     // Fire up a server.
-    let mut server = ServerImpl { fmc };
+    let rx_packet = RX_PACKET.claim();
```

Comment:
What's the intuition for when we should use `ClaimOnceCell` versus `mutable_statics!`?

Reply:
`mutable_statics!` leaves its backing memory uninitialized, so it can be filled in with things that aren't legal consts. This may also provide a benefit in cases where the initializer isn't zeroes but the initialization code is relatively simple (because a non-zeroes const initializer ends up fully expanded in the data image).

 `ClaimOnceCell` requires a legal const in its initializer but has lower complexity (no macros) and runtime cost (no initialization).